zephyr/lib/cpu.c

```diff
@@ -57,71 +57,7 @@ static FUNC_NORETURN void secondary_init(void *arg)
 	CODE_UNREACHABLE; /* LCOV_EXCL_LINE */
 }
 
-#ifndef CONFIG_ACE
```

Comment:
would the diff be smaller if you changed this to `#if !CONFIG_ZEPHYR_NATIVE_DRIVERS` and moved the added code here? With this kind of a change it's difficult to see whether the code id changed or just moved.

Reply:
yeah probably. I personally dont like the !CONFIGs as they are really "easy" to misinterpret for not having the ! when reading the code. So I would prefer to have the configs without ! if not absolutely needed. But yes, you are also right that this PR didnt change the code of the other branch at all, just moved it around.